src/baldr/graphtile.cc

```diff
@@ -176,32 +180,33 @@ GraphTile::GraphTile(const std::string& tile_dir,
 
 GraphTile::GraphTile() = default;
 
-void GraphTile::SaveTileToFile(const std::vector<char>& tile_data, const std::string& disk_location) {
+void GraphTile::SaveTileToFile(const std::vector<char>& tile_data,
```

Comment:
this method isn't used anymore it seems. it was introduced in https://github.com/valhalla/valhalla/pull/2026 for `GraphTile::CacheTileURL` but that code was ripped out at some point later. might as well remove it?